docs/topics/spiders.rst

```diff
@@ -826,3 +832,29 @@ Combine SitemapSpider with other sources of urls::
 .. _robots.txt: https://www.robotstxt.org/
 .. _TLD: https://en.wikipedia.org/wiki/Top-level_domain
 .. _Scrapyd documentation: https://scrapyd.readthedocs.io/en/latest/
+
+
+Base spiders
+============
+
+Base spiders are :class:`~scrapy.spiders.Spider` subclasses that are not meant
+to be run by Scrapy. They are only meant to be subclassed to create regular
+spiders or other base spiders. They are one way to share code between two or
+more spiders.
+
+Use the :func:`~scrapy.spiders.ignore_spider` decorator to mark any base spider
+class:
+
+.. autodecorator:: scrapy.spiders.ignore_spider
+
+For example::
+
+    from scrapy.spiders import ignore_spider, Spider
+
+    @ignore_spider
+    class MyBaseSpider(Spider):
+        pass
+
+If :setting:`SPIDER_LOADER_REQUIRE_NAME` is ``True`` (default), any
```

Comment:
I woder if default True would be confusing here - does it require more elaboration? For new users default is going to be False, right?

Reply:
I think removing “(default)” altogether will make users open the setting link, and there they will be able to find the details about it.